core/src/main/java/org/jivesoftware/spark/ui/login/CertificatesManagerSettingsPanel.java

```diff
@@ -46,7 +47,7 @@
 	private final static Insets DEFAULT_INSETS = new Insets(5, 5, 5, 5);
 	private final LocalPreferences localPreferences;
 	private CertificateController certControll;
-	private JTable certTable = new JTable();
+	private static JTable certTable = new JTable();
```

Comment:
Now I left it as is but if you still feel that I should change it I will come back to this.